### PR TITLE
5806 Unable to go to Name Request site from Create Account

### PR DESCRIPTION
**Pull request by vysakh-menon-aot**
*Issue #:* /bcgov/entity#5806

*Description of changes:*
 - NAME_REQUEST_URL and NRO_URL required for sbc-auth to redirect back after creation of a new account (This is required for MVP)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

**Review by cameron-freshworks — APPROVED** [review]
Very cool, looks good! Just need to make sure this value is configured in Dev/Test/Prod.
Also add this dependency to our Name Request Release ticket please :) #5732

File: client/src/plugins/getConfig.ts [hunk 1]
```diff
@@ -58,6 +58,9 @@ export function getConfig (): Promise<EnvConfigI> {
     const legalApiUrl = response.data[0]['LEGAL_API_URL']
     sessionStorage.setItem('LEGAL_API_URL', legalApiUrl)
 
+    const nameRequestUrl = response.data[0]['NAME_REQUEST_URL']
+    sessionStorage.setItem('NAME_REQUEST_URL', nameRequestUrl)
```
Comment (severinbeauvais) [inline client/src/plugins/getConfig.ts:62]: What is this used for?

File: client/src/plugins/getConfig.ts [hunk 2]
```diff
@@ -65,9 +65,13 @@ export function getConfig (): Promise<EnvConfigI> {
...
      * This will be removed when a fix is made to sbc-common-components to handle this
      */
+    const nameRequestUrl = response.data[0]['NAME_REQUEST_URL']
+    const nroUrl = response.data[0]['NRO_URL']
     const authConfig = {
       'AUTH_URL': businessUrl,
-      'VUE_APP_PAY_ROOT_API': authPayUrl
+      'VUE_APP_PAY_ROOT_API': authPayUrl,
+      'NAME_REQUEST_URL': nameRequestUrl,
+      'NRO_URL': nroUrl
```
Comment (severinbeauvais) [inline client/src/plugins/getConfig.ts:74]: Why is this needed? Doesn't auth-web read its own config?

Comment (vysakh-menon-aot) [reply client/src/plugins/getConfig.ts:74]: There is a bug in sbc-auth, discussed with Saravan. See the comment https://github.com/bcgov/entity/issues/5806#issuecomment-742675724

**Review by severinbeauvais — APPROVED** [review]
